Api/Modpacks.cs

```diff
@@ -357,7 +358,8 @@ static class Modpacks {
 
             if ((string)btn.Tag == "btn1") {
                 OpenFileDialog ofd = new OpenFileDialog {
-                    InitialDirectory = "::{20D04FE0-3AEA-1069-A2D8-08002B30309D}"  // using the GUID to access 'This PC' folder
+                    InitialDirectory = "::{20D04FE0-3AEA-1069-A2D8-08002B30309D}",  // using the GUID to access 'This PC' folder
+                    Title = "Select a new mod file to apply to your game:"
```

Comment:
Similarly to the previous comment, lets change this to "Select a mod file" (since you an only select one here)

Reply:
Done